test/check-machines-lifecycle

```diff
@@ -430,6 +430,26 @@ class TestMachinesLifecycle(VirtualMachinesCase):
             qemu_img_info = m.execute(f"qemu-img info /var/lib/libvirt/images/{name}-2.img")
             self.assertNotIn("snapshotA", qemu_img_info)
 
+            # Snapshot deletion was failing for VMs with file-directory storage-pool volumes as disks.
+            # It was fixed since libvirt 7.1.0 at https://gitlab.com/libvirt/libvirt/-/commit/6423e3082835db932dd285f3ae75b5dad9d91c83
+            if m.execute("virsh --version") >= "7.1.0":
```

Comment:
When we add conditions like this we always have this else which will notify us when the condition is not needed anymore. 

Since we know it's only ubuntu-stable that's reaches this else, what do you think if we would do something like:
else:
   assertEqual(m.image, "ubuntu-stable")